test/core/derivatives/implementation/base.py

```diff
@@ -26,7 +26,7 @@ def jac_mat_prod(self, mat: Tensor) -> Tensor:
             Tensor representing the result of Jacobian-vector product.
                 product[v] = J @ mat[v]
         """
-        raise NotImplementedError
+        return
```

Comment:
Is this the best approach?
I'm satisfied with the solution. However, I'm afraid type checkers that will be added later will complain that it is no `Tensor`

Reply:
This solution makes the docstring checkers pass without complaining about `missing raises`, `excessive return`. I currently don't know a better way to achieve this, even though it might break type checkers. Would have to read up on it.

Reply:
I found that the `return` can be replaced with `raise NotImplementedError` from [darglint>=1.8.0](https://pyup.io/changelogs/darglint/) on. Locally, I am on `1.6`, will upgrade, and raise `NotImplementedError`s.